docs/src/pages/guides/api.md

```diff
@@ -81,12 +81,12 @@ type Props = {
 };
 ```
 
-- Option 2 *enum*: `<Button>`, `<Button type="raised">`, `<Button type="fab">`.
+- Option 2 *enum*: `<Button>`, `<Button variant="raised">`, `<Button variant="fab">`.
   With this API, you prevent invalid combination from being used, you bound the number of properties you expose, and you can easily support new values in the future.
 
 ```tsx
 type Props = {
-  type: 'flat' | 'raised' | 'fab';
+  variant: 'flat' | 'raised' | 'fab';
```

Comment:
`type` is already a native property on the input and button component.
We would rather avoid the conflict.